public class RightAlignedNumberTriangle {
    public static void rightAlignedNumberTriangle(int n, int count) {
        for (int i = 1; i <= n; i++) {
            int spaces = n - i;
            for (int j = 1; j <= spaces; j++) {
                System.out.print(" ");
            }
            for (int j = 1; j <= i; j++) {
                System.out.print(count + " ");
                count++;
            }
            System.out.println();
        }
    }

    public static void main(String[] args) {
        rightAlignedNumberTriangle(4, 1);
    }
}
